watcher_pods.go

```diff
@@ -33,7 +33,7 @@ func handlePodTerminationEvent(ctx context.Context, containerStatus *v1.Containe
 	}
 
 	setTagIfNotEmpty(scope, "reason", state.Reason)
-	setTagIfNotEmpty(scope, "kind", pod.Kind)
+	setTagIfNotEmpty(scope, "kind", "Pod")
```

Comment:
If there's no constants in client-go for resource kinds, let's declare those ourselves (e.g. will help to avoid typos).